.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "prettier.prettierPath": "./node_modules/prettier",
+  "prettier.prettierPath": "",
```

Comment:
Apparently the `vscode` plugin for prettier v3 now automatically finds the prettier package.